extensions/blocks/membership-button/view.scss

```diff
@@ -0,0 +1,55 @@
+/* Additional styling to thickbox that displays modal */
+/* stylelint-disable selector-max-id */
+
+// @TODO: Replace with Gutenberg variables
+$--muriel-gray-0: #f6f6f6;
+$--muriel-gray-50: #e1e2e2;
+$--muriel-blue-200: #95adc1;
+$--muriel-hot-blue-500: #005fb7;
+
+.jetpack-memberships-modal #TB_title {
+	border-radius: 4px 4px 0 0;
+}
+#TB_window.jetpack-memberships-modal {
+	border-radius: 4px;
+	background-color: $--muriel-gray-0;
```

Comment:
Seems like in Jetpack these kind of SASS variables for Muriel colours work nowadays without needing to define them on top of the file: `$muriel-gray-0`.

(CSS variables like `--muriel-gray-0` used to work but looks like that's no more the case)

Reply:
Oh seems like `var( --muriel-gray-0 )` might work. We'll have to document this stuff better. :-)